dashboard-for-desktop/articles/dashboard-designer/provide-data/supported-data-sources.md

```diff
@@ -118,9 +116,23 @@ To create a connection to an EF data source, see [Connect to EF Data Sources](co
 * [SQL Data Source](#sql-data-source)
```

Comment:
1) Line 105 (above): Use the available data context to connect to EF data source.

Can we update this to the following?

`Use the available data context to connect to an EF data source.`

2) Line 114: Federated data source supports the following data source types:

Can we update this to:

`A Federated data source supports the following data source types:`